video/vdu.h

```diff
@@ -185,13 +185,8 @@ void VDUStreamProcessor::vdu_plot() {
 			// move to modes
 			moveTo();
 			break;
-		case 2:
-		case 6:
-			// draw inverse logical colour not supported
-			debug_log("plot inverse logical colour not implemented\n\r");
-			break;
```

Comment:
this code here was trapping the "inverse" plot modes and preventing them from working

in reality, only the `plotLine` calls in the `default` block below actually support "inverse" or "NOT" plotting.

it _may_ be possible to add support for an "inverse" version of `plotPoint`.  the current code uses `canvas->setPixel`, but that doesn't support NOT, but it _could_ be replaced with a `canvas->moveTo` followed by a `canvas->lineTo` to draw a single pixel line that _would_ support NOT

this still leaves all of the "filled shape" and circle plot operations that cannot support NOT with the current fab-gl

I'm not sure yet whether "disable inverse operations (if it's not possible)" is the right way to go (similar to the previous approach), or if the currently implemented "do inverse if you can, otherwise just draw in fg colour" is acceptable